.travis.yml

```diff
@@ -13,7 +13,7 @@ before_script:
   - export DJANGO_SECRET_KEY=foobarbaz
 script:
   - cd vms
-  - python manage.py syncdb --noinput
+  - python manage.py makemigrations auth volunteer administrator organization event job shift registration
```

Comment:
if we are pushing migrations, the pipeline script should just apply it - and not make it.

Reply:
@tonythomas01 We are not pushing migrations. They are created on this command.

Reply:
again a question - why do we have to say explicitly which all to migrate ? if there are extensions with migrations - this will break ?

Reply:
by extensions I meant additional apps that VMS has installed (like in INSTALLED_APPS). Usually some of these apps have migrations, and they are migrated when you do `migrate` without arguments. Said that, if the thing works perfectly without explicitly mentioning the app names - please go for it. 

Also, can you point me to this doc ?

Reply:
It works fine without breaking for any installed apps.

Reply:
Perfect. also, please create an issue for fixing this for later (so someone might figure out why exactly we need to pass app names explicitly).